packages/atomic/src/components/recommendations/atomic-recs-interface/atomic-recs-interface.tsx

```diff
@@ -140,6 +141,24 @@ export class AtomicRecsInterface
    */
   @Prop({reflect: true}) public iconAssetsPath = './assets';
 
+  /**
+   * The value to set the [nonce](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce) attribute to on inline script and style elements generated by this interface and its child components.
+   * If your application is served with a Content Security Policy (CSP) that doesn't include the `script-src: 'unsafe-inline'` or `style-src: 'unsafe-inline'` directives,
+   * you should ensure that your application server generates a new nonce on every page load and uses the generated value to set this prop and serve the corresponding CSP response headers
+   * (i.e., script-src 'nonce-<YOUR_GENERATED_NONCE>' and style-src 'nonce-<YOUR_GENERATED_NONCE>').
+   * Otherwise you may see console errors such as
+   *  - Refused to execute inline script because it violates the following Content Security Policy directive: [...]
+   *  - Refused to apply inline style because it violates the following Content Security Policy directive: [...].
+   * @example:
```

Comment:
I think we should add

> When using a nonce, the first import of Atomic should be to import & call the `setNonce` function with the generated nonce value.

In the documentation each time this prop is exposed on a component.

Reply:
recs was not ported, that was some artifacts